storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -41,7 +41,13 @@ After full migration, in-tree plugins will eventually be removed in future versi
 
 For information about dynamically provisioning AWS EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store].
 
+ifndef::openshift-rosa,openshift-dedicated[]
+include::modules/persistent-storage-byok.adoc[leveloffset=+1]
+
```

Comment:
```suggestion

[NOTE]
====
If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in storage class, AWS EBS CSI driver uses the AWS managed(alias/aws/ebs which is created by Amazon EBS automatically in each Region) by default to encrypt provisioned storage volumes. In addition, the managed storage classes are all with `encrypted: "true"` setting.
====

```
We'd better also add note for ebs csi driver behavior when the not set the customer kms key.
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html -> Default KMS key for EBS encryption

@chao007 Does gcp also need add some note for the driver behavior when the not set the customer kms key?

Reply:
Done. Just waiting on @chao007 for GCP possible note.

Reply:
Thanks for the update! 
LGTM for the AWS EBS BYOK part

Reply:
GCP will use [google managed encrypted key](https://cloud.google.com/storage/docs/encryption/default-keys) when no customer key set in storageclass. I think we don't need to clarify in our doc.

Reply:
Sure, thanks for the double confirm!

Reply:
Ok. Thanks for update!